src/OmniSharp.Abstractions/Roslyn/OmniSharpWorkspace.cs

```diff
@@ -13,6 +17,10 @@ public class OmnisharpWorkspace : Workspace
 
         public BufferManager BufferManager { get; private set; }
 
+        public CompositionHost PluginHost { get; private set; }
+
+        public static OmnisharpWorkspace Instance{ get; private set; }
```

Comment:
I don't like having to have a static property.  Does anyone have any idea how to insert an export into an MEF CompositionHost?

I want to force the value in from the startup class if possible.